### PR TITLE
Create next.py fixes #1327

### LINKED ISSUE #1327: implement !next (a.k.a ship it)
No description given.
Issue created by jholtom!uid156752@gateway/web/irccloud.com/x-swgowrxvkuqcjqzg

### PR DESCRIPTION
**Pull request by jholtom**

### CODE REVIEW

**Comment by pefoley2** [comment]
I'll take a look when I'm less absurdly busy.

**Comment by jholtom** [comment]
I know how to code, I'll fix it sooner or later :P

File: cslbot/commands/next.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+@Command('next')
+def cmd(send, msg, args):
+    """Ships a product.
+
+    Syntax: {command}
+
+    """
+    send("NEXT! "+ random.choice(squirrels))
+
```
Comment (pefoley2) [inline cslbot/commands/next.py]: Codacy Issue found: [Trailing whitespace](https://www.codacy.com/app/pefoley2/cslbot/file/2384308182/issues/source?bid=3176156&fileBranchId=3227377#l44)

File: cslbot/commands/next.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+def cmd(send, msg, args):
+    """Ships a product.
+
+    Syntax: {command}
+
+    """
+    send("NEXT! "+ random.choice(squirrels))
+    
+@Command('shipit')
+def cmd(send, msg, args):
```
Comment (pefoley2) [inline cslbot/commands/next.py]: Codacy Issue found: [function already defined line 38](https://www.codacy.com/app/pefoley2/cslbot/file/2384308182/issues/source?bid=3176156&fileBranchId=3227377#l46)

File: cslbot/commands/next.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  "http://img70.imageshack.us/img70/4853/cutesquirrels27rn9.jpg",
+  "http://img70.imageshack.us/img70/9615/cutesquirrels15ac7.jpg",
+  "https://dl.dropboxusercontent.com/u/602885/github/sniper-squirrel.jpg",
+  "http://1.bp.blogspot.com/_v0neUj-VDa4/TFBEbqFQcII/AAAAAAAAFBU/E8kPNmF1h1E/s640/squirrelbacca-thumb.jpg",
+  "https://dl.dropboxusercontent.com/u/602885/github/soldier-squirrel.jpg",
+  "https://dl.dropboxusercontent.com/u/602885/github/squirrelmobster.jpeg",
+]
+
+@Command('next')
+def cmd(send, msg, args):
```
Comment (pefoley2) [inline cslbot/commands/next.py]: Codacy Issue found: [Unused argument 'args'](https://www.codacy.com/app/pefoley2/cslbot/file/2384308182/issues/source?bid=3176156&fileBranchId=3227377#l37)

**Pull request merged**